src/views/SpaceCreate.vue

```diff
@@ -118,24 +135,15 @@ const disableChoiceEdit = computed(() => form.value.type === 'basic');
 
 function addChoice(num) {
   for (let i = 1; i <= num; i++) {
-    counter.value++;
-    choices.value.push({ key: counter.value, text: '' });
-  }
-}
-
-function removeChoice(i) {
-  choices.value.splice(i, 1);
-}
-
-function setDate(ts) {
-  if (selectedDate.value) {
-    form.value[selectedDate.value] = ts;
+    choices.value.push({ id: choices.value.length, text: '' });
```

Comment:
This function feels a bit unnecessary. There's just one place where it's used with num = 2. We can just call the function twice. This confused me a bit. I'd either remove it or rename it to `addChoices`. :D

Reply:
You did? I meant the function. And I'd actually just remove it entirely and just push to choices directly where this function was used. I don't see much value in this num parameter and a for loop.

```
addChoice(2)
```
vs:
```
choices.value.push({ id: choices.value.length, text: '' });
choices.value.push({ id: choices.value.length, text: '' });
```

or

`@click="addChoice(1)"` vs `@click="choices.push({ id: choices.length, text: '' });"`

I think that's just much clearer. Or am I missing something?